parteh/PRTParamsFATESMod.F90

```diff
@@ -1526,18 +1698,29 @@ subroutine PRTCheckParams(is_master)
            
            ! Check that branch turnover doesn't exeed 1 day
            if ( (years_per_day / prt_params%branch_long(ipft)) > 1._r8 ) then
+              write(fates_log(),*) "---~---"
               write(fates_log(),*) 'Branch turnover time-scale is greater than 1 day!'
               write(fates_log(),*) 'ipft: ',ipft
               write(fates_log(),*) 'branch_long(ipft): ',prt_params%branch_long(ipft),' [years]'
-              write(fates_log(),*) 'Aborting'
-              call endrun(msg=errMsg(sourcefile, __LINE__))
+              write(fates_log(),*) "---~---"
+              write(fates_log(),*) ''
+              write(fates_log(),*) ''
+              nerror = nerror + 1
            end if
         end if
 
 
      end do pftloop
 
 
+     ! If any error was found, abort. We add a single point to abort the run after all
+     ! checks so users can get all the errors and address them in one go (as opposed to
+     ! multiple submissions).
+     if (nerror > 0) then
+        write(fates_log(),*) 'One or more parameter errors found. Aborting.'
+        call endrun(msg=errMsg(sourcefile, __LINE__))
+     end if
```

Comment:
double checked to make sure that all checks removed endrun and added nerror, looks good